addons/mrp_subcontracting/models/stock_warehouse.py

```diff
@@ -26,6 +26,16 @@ class StockWarehouse(models.Model):
         'stock.picking.type', 'Subcontracting Resupply Operation Type',
         domain=[('code', '=', 'outgoing')])
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        self._update_subcontracting_locations_rules()
+        return res
```

Comment:
I'm not sure if this is working correctly. When I tested by making a new subcontracting location and then creating a new Warehouse, the new Warehouse doesn't seem to have the rules added for the subcontracting location :thinking: